### PR TITLE
#MySQL Task. TapeGhad

### PR DESCRIPTION
**Pull request by TapeGhad**

### CODE REVIEW

**Comment by TapeGhad** [comment]
image
I suppose, something wrong with northwind db in travis

**Comment by TapeGhad** [comment]
image

PC tests are fine

**Comment by TapeGhad** [comment]
Fixed :) 
Travis is sensetive to camelcase

File: task/sql-tasks.js [hunk 1]
```diff
@@ -129,7 +195,14 @@ async function task_1_8(db) {
  *
  */
 async function task_1_9(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            CustomerID,
+            ContactName
+        FROM Customers
+        WHERE ContactName REGEXP '^[F][^\.]{2}[n]'
```
Comment (AlexMokin) [inline task/sql-tasks.js]: Use regexp only if not possible to use **like**. In this case it's possible to replace it with like

File: task/sql-tasks.js [hunk 2]
```diff
@@ -177,7 +274,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            COUNT(*) AS 'TotalOfCurrentProducts',
+            SUM(Discontinued) AS 'TotalOfDiscontinuedProducts'
```
Comment (AlexMokin) [inline task/sql-tasks.js]: use count to show intent explicitly

in this case it's better to just use 2 select subqueries

from-clause is optional.

 it's better to write this query like this

```
select
   (select count()...) as 'T'
   (select count()...) as 'TT'
```

File: task/sql-tasks.js [hunk 3]
```diff
@@ -199,7 +310,24 @@ async function task_1_14(db) {
  *
  */
 async function task_1_15(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            SUM(MONTH(OrderDate) = 1) AS 'January',
```
Comment (AlexMokin) [inline task/sql-tasks.js]: use count to show intent explicitly

**Review by AlexMokin — COMMENTED** [review]
Looks good. See comments.
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

Also please provide link to a test resutls

**Comment by TapeGhad** [comment]
https://travis-ci.com/github/AisBrestEDU/db-assignments/builds/181846938

**Comment by TapeGhad** [comment]
Fixed

**Comment by AlexMokin** [comment]
accepted

**Pull request closed without merging**